packages/app/src/ExplorerTabs.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from 'react';
+import { FiList, FiSearch } from 'react-icons/fi';
+
+import styles from './ExplorerTabs.module.css';
+import Explorer from './explorer/Explorer';
+import { useDataContext } from './providers/DataProvider';
+import SearchBoxContainer from './search/SearchTabContainer';
```

Comment:
Inconsistent file name here, though I'd probably rename this component `SearchContainer`, and its child `Search`. The term `SearchTab` is not quite right, since the "tab" is the element you click on -- it's technically a "panel" -- but I don't think it's worth mentioning in the component name, since it doesn't contain any code that makes it a tab panel specifically.